pkg/core/disk.go

```diff
@@ -58,11 +60,32 @@ func (d *DiskOption) Validate() error {
 			return fmt.Errorf("unknown units of size : %s, DiskOption : %v", d.Size, d)
 		}
 	}
+
 	if d.Action == DiskFillAction {
 		if d.FillByFallocate && byteSize == 0 {
 			return fmt.Errorf("fallocate not suppurt 0 size or 0 percent data, "+
 				"if you want allocate a 0 size file please set fallocate=false, DiskOption : %v", d)
 		}
+
+		_, err := os.Stat(d.Path)
+		if err != nil {
+			if os.IsNotExist(err) {
+				// check if Path of file is valid when Path is not empty
+				if d.Path != "" {
+					var b []byte
+					if err := ioutil.WriteFile(d.Path, b, 0644); err != nil {
+						return err
+					}
+					if err := os.Remove(d.Path); err != nil {
+						return err
+					}
```

Comment:
why write something and then remove it?

Reply:
This is a empirical way to check if a path is valid.
[url](https://stackoverflow.com/questions/35231846/golang-check-if-string-is-valid-path)

Reply:
maybe we can ignore check here, and after run with this invalid path, dd/fallocate will  report the error message

Reply:
I think in future users may combine some attacks with our scheduler, if they apply some attacks like:first -> network attack for a day, next -> disk attack for a day, than check invalid path in validation part will bring convenience to found problem ahead of long time executing. And another problem is the error message of dd or fallocate sometimes may bring confuse to user.